### PR TITLE
Change MouseClickNode to calculate x,y from GetContentQuads instead of GetBoxModel.

### LINKED ISSUE #435: line-height breaks click on even number lines
#### What versions are you running?

<pre>
$ go list -m github.com/chromedp/chromedp
github.com/chromedp/chromedp v0.0.0-20190617065505-d55cf9043e05
$ google-chrome --version
Version 75.0.3770.100 (Official Build) (64-bit)
$ go version
go version go1.12.1 darwin/amd64
</pre>

#### What did you do? Include clear steps.

Example above shows, that click in the middle of a block will do nothing, since 0.4 in the middle is not clickable.

```html
<html>
<body>
<div style="width: 100px">
    <a href="#" style="line-height: 1.4;">long link long link long link</a>
</div>
</body>
</html>
```

#### What did you expect to see?

Click

#### What did you see instead?

No click

### PR DESCRIPTION
**Pull request by ZzEeKkAa**
fixes #435

### CODE REVIEW

File: input_test.go [hunk 1]
```diff
@@ -69,25 +69,27 @@ func TestMouseClickNode(t *testing.T) {
...
-		{`input3`, "bar-right", Button("right"), ByID},
-		{`document.querySelector('#input3')`, "bar-right", ButtonType(input.ButtonRight), ByJSPath},
+		{"input.html", `button2`, "foo", ButtonType(input.ButtonNone), ByID},
+		{"input.html", `button2`, "bar", ButtonType(input.ButtonLeft), ByID},
+		{"input.html", `button2`, "bar-middle", ButtonType(input.ButtonMiddle), ByID},
+		{"input.html", `input3`, "foo", ButtonModifiers(input.ModifierNone), ByID},
+		{"input.html", `input3`, "bar-right", ButtonType(input.ButtonRight), ByID},
+		{"input.html", `input3`, "bar-right", Button("right"), ByID},
+		{"input.html", `document.querySelector('#input3')`, "bar-right", ButtonType(input.ButtonRight), ByJSPath},
+		{"click.html", `link`, "clicked", ButtonType(input.ButtonLeft), ByID},
```
Comment (mvdan) [inline input_test.go]: why not add the extra HTML to `input.html`? After all, this is `input_test.go`, and it will keep the test more consistent.

**Comment by mvdan** [comment]
Friendly ping @ZzEeKkAa - do you plan on updating this PR?

File: input_test.go (hunk 1 above)
Comment (ZzEeKkAa) [reply input_test.go]: @mvdan changed to input.html.

**Comment by ZzEeKkAa** [comment]
Thanks for pinging! Done.

**Pull request merged**